src/System/include/BipedalLocomotion/System/FixStepIntegrator.h

```diff
@@ -0,0 +1,73 @@
+/**
+ * @file FixStepIntegrator.h
+ * @authors Giulio Romualdi
+ * @copyright 2020 Istituto Italiano di Tecnologia (IIT). This software may be modified and
+ * distributed under the terms of the GNU Lesser General Public License v2.1 or any later version.
+ */
+
+#ifndef BIPEDAL_LOCOMOTION_SYSTEM_FIX_STEP_INTEGRATOR_H
+#define BIPEDAL_LOCOMOTION_SYSTEM_FIX_STEP_INTEGRATOR_H
+
+#include <memory>
+#include <tuple>
+
+#include <BipedalLocomotion/System/Integrator.h>
+
+namespace BipedalLocomotion
+{
+namespace System
+{
+
+/**
+ * Fixed step integrator base class. Please inherit publicly from this class in order to specify a
+ * custom integration method.
+ * @tparam DynamicalSystemDerived a class derived from DynamicalSystem
+ */
+template <typename DynamicalSystemDerived>
+class FixStepIntegrator : public Integrator<DynamicalSystemDerived>
```

Comment:
Does it make sense to inherit from ``Advanceable`` too?

Reply:
By further thinking about it and after a quick F2F it appeared it would make more sense to have an object including both a dynamical system and a fixed step integrator as ``Advanceable``, but this is out of scope.